src/components/time-picker/time-picker.tsx

```diff
@@ -0,0 +1,374 @@
+import * as keycode from 'keycode';
+import * as React from 'react';
+import {SBComponent} from 'stylable-react-component';
+import {Stepper} from '../stepper';
+import styles from './time-picker.st.css';
+import {
+    Ampm, ampmLabels, Format,
+    formatTimeChunk, getCircularValue, is12TimeFormat, isTimeSegment,
+    isTouch, isValidValue, Segment, TimeSegment, to24, toAmpm
```

Comment:
no, this one should not. It is very specific to this component. I would rather call it `isNative`